cranelift/codegen/src/machinst/isle.rs

```diff
@@ -806,7 +806,7 @@ macro_rules! isle_prelude_methods {
 
         #[inline]
         fn emit_u128_le_const(&mut self, value: u128) -> VCodeConstant {
-            let data = VCodeConstantData::Generated(value.to_le_bytes().iter().cloned().collect());
+            let data = VCodeConstantData::Generated(value.to_le_bytes().iter().copied().collect());
```

Comment:
Oh, after Chris' suggestion, now you can simplify this to `value.to_le_bytes().into()`, I think.

Reply:
I guess `ConstantData` doesn't have a `From` implementation for arrays, but `.as_slice().into()` ought to work at least.

Reply:
Looks like we're stuck with my verbose monstrosity:
```
809 |             let data = VCodeConstantData::Generated(value.to_le_bytes().iter().into());
    |                                                                                ^^^^ the trait `From<core::slice::Iter<'_, u8>>` is not implemented for `ConstantData`
```

Reply:
The problem there is the `.iter()`.

Reply:
Yeah, I get a similar error without it. I had been experimenting :)
```
809 |             let data = VCodeConstantData::Generated(value.to_le_bytes().into());
    |                                                                         ^^^^ the trait `From<[u8; 16]>` is not implemented for `ConstantData`
```

Reply:
I'd figured out that just `.into()` wouldn't work after I posted my first comment. Did you try my second suggestion, of `.as_slice().into()`?

Reply:
Hooray! It'd probably be nice if `ConstantData` actually had a `From<[T; n]>` implementation too, but I didn't want to suggest doing that in this PR. :sweat_smile: